cypress/support/commands.js

```diff
@@ -23,3 +23,28 @@
 //
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+
+const { generateUser } = require("./generate");
+
+Cypress.Commands.add('findById', (id) => {
+    cy.get(`#${id}`);
+})
+
+Cypress.Commands.add('registerNewUser', () => {
```

Comment:
you can modify that function. For example when you register the user, you can take tokwn from a response and set it, this will help you to avoid using additional code in every it()